guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -147,16 +147,12 @@ endif::[]
 ----
 # {foreman-maintain} service restart
 ----
-. Optional: If you made manual edits to DNS or DHCP configuration files, check and restore any changes required to the DNS and DHCP configuration files using the backups that you make.
```

Comment:
Is this no longer necessary? How does it relate to pulp3?

Reply:
@ianf77 please make a call here

Reply:
It isn't related to Pulp 3.

In thee original PR#663

 ianf77 22 days ago Author Member

Steps 8 and 9 in the foreman documentation are identical to Steps 10 and 11, should the steps be repeated, or is it an error?
@melcorr melcorr 22 days ago Member

I see now. They were added in error here: https://github.com/theforeman/foreman-documentation/pull/490/files#diff-c24ec779d7504e0ce3bb6f2834002cdb6abff1023e249d73704797c54ac4a7d2R134
Resolving this, thanks!